src/Chainweb/CutDB.hs

```diff
@@ -146,6 +148,7 @@ defaultCutDbConfig v = CutDbConfig
 data CutDb cas = CutDb
     { _cutDbCut :: !(TVar Cut)
     , _cutDbQueue :: !(PQueue (Down CutHashes))
+    , _cutNetworkCutHeight :: !(TVar (Maybe BlockHeight))
```

Comment:
`Maybe`, so that anyone listening to this `TVar` can avoid special logic during initial network conditions (i.e. before any `Cut` information has come in).

Reply:
Would `TMVar` work here?